docs/reference/searchable-snapshots/index.asciidoc

```diff
@@ -120,39 +120,38 @@ To search a snapshot, you must first mount it locally as an index. Usually
 are two options for mounting a snapshot, each with different performance
 characteristics and local storage footprints:
 
-[[full-copy]]
-Full copy::
+[[fully-mounted]]
```

Comment:
As a general note, I think this page could use a refactor to reorder the content and deduplicate. However, that work is less pressing and outside the scope of this PR. I opened https://github.com/elastic/elasticsearch/issues/72728 to track.